metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage.py

```diff
@@ -183,6 +205,49 @@ def _get_upstream_lineage_info(
 
         assert self._lineage_map is not None
```

Comment:
Minor: is this assert safe? Why should we assume that these maps exist?

Reply:
Removing these asserts and initializing the variables as defaultdict instead of optional dict. We do not need to differentiate between first run and subsequent runs anymore. This was required earlier for caching/reusing as these methods were called in loop during table extraction phase.